server/config/disaster_dashboard/Earth.textproto

```diff
@@ -152,41 +173,47 @@ metadata {
       color: "#304ffe"
       default_severity_filter: {
         prop: "differenceTemperature"
+        display_name: "Difference"
         unit: "Celsius"
         # If updating the upper_limit, also update the Extreme Cold event description (below, in this file).
         upper_limit: -18
         lower_limit: -35
       }
+      display_prop {
+        prop: "referenceTemperature"
+        display_name: "Reference"
+        unit: "Celsius"
+      }
     }
   }
 }
 
-categories {
-  title: "Severe Weather and Natural Disasters"
+# categories {
```

Comment:
curious why we're removing the main map?

Reply:
ditto - removed to make debugging easier but has since been reverted